modules/apps/jaggery/mdm/src/mdm/modules/user.js

```diff
@@ -420,25 +437,12 @@ var userModule = function () {
         }
         try {
             utility.startTenantFlow(carbonUser);
-
+            log.error("here")
```

Comment:
This was a fault due to that all the committed changes has been reverted and all the local changes has been re based.Issue has been fixed as soon as encountered.